lib/etl/acknowledger.ex

```diff
@@ -0,0 +1,3 @@
+defmodule Etl.Acknowledger do
+  @callback ack(ack_ref :: term, success :: [Etl.Message.t()], fail :: [Etl.Message.t()]) :: :ok
+end
```

Comment:
I wanted to use Mox to test `Etl.ack/1` so I needed this behaviour.

Reply:
probably value in allowing for a custom acknowledger for crazy people like us who aren't satisfied with what comes out of the box.